app/javascript/src/utils/index.js

```diff
@@ -1,3 +1,5 @@
 import * as R from "ramda";
 
+export * from "./time";
```

Comment:
What is the point of this line???

Reply:
I used it to export `timeAgoInWords` functions. But I don't think any more functions would be added to the `time.js` file. So I will move the `timeAgoInWords` to the `index.js` file.